client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,43 @@
+.spub-header-component {
+	border-bottom:	border-bottom: 1px solid #d2d2d2;
+;
+	padding: 200px;
+
+	h1 {
+		margin: 0;
+
+		font-size: 40px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 56px;
+		letter-spacing: 0em;
+		text-align: left;
+
+		box-decoration-break: clone;
+	}
+
+	.tabs{
+		color: var(--community-accent-dark);
+		max-width: 900px;
```

Comment:
What is this value for?

Reply:
its suppose to change the color of the tabs, but how that is being handled has been updated